.github/CODEOWNERS

```diff
@@ -936,6 +936,9 @@
 # PRLabel: %Communication - Resource Manager
 /sdk/communication/Azure.ResourceManager.Communication/            @archerzz @ArcturusZhang @ArthurMa1978
 
+# PRLabel: %Network - Front Door
```

Comment:
This isn't correct.   Please refer to my earlier comment for what form this entry needs to take.